tests/test_ssa_value.py

```diff
@@ -37,13 +37,8 @@ def test_var_mixed_builder():
         _ = b.get(op)
 
 
-@pytest.mark.parametrize("name,result", [
-    ('a', 'a'),
-    ('test', 'test'),
-    ('test1', None),
-    ('1', None),
-])
-def test_ssa_value_name_hints(name, result):
+@pytest.mark.parametrize("name", ["a", "test", "1", "12", "-2", "test_123"])
```

Comment:
-2 ?! Everyday I'm more surprised by MLIR.
Also, can we change the documentation of this test then?

Reply:
According to [this](https://mlir.llvm.org/docs/LangRef/#identifiers-and-keywords) document, dashes are indeed allowed characters for ssa value names.

I don't like it either.

Reply:
kebab case names FTW

Reply:
Yeah I checked it yesterday, and `-2` was allowed.
What I meant was the documentation of this test, because it is now outdated!